app/views/cases/show.html.slim

```diff
@@ -47,7 +47,7 @@ div id="case-#{@case.id}"
     = render partial: 'cases/case_status', locals: {case_details: @case}
 
   section.case-info
-    = render partial: 'cases/linked_cases', locals: { case_details: @case }
+    = render partial: 'cases/linked_cases', locals: { case_details: @case, permitted_events: @permitted_events }
```

Comment:
Don't need `permitted_events` as checking the policy is handled inside the [helper used](https://github.com/ministryofjustice/correspondence_tool_staff/blob/d2d7f53d40168042daad79656662d1aa7e2e2c97/app/helpers/cases_helper.rb#L145) by `_linked_cases` partial.